api/swagger.yaml

```diff
@@ -628,39 +628,59 @@ paths:
           description: Internal Server Error
           schema:
             $ref: '#/definitions/Error'
-  /boot/v1/service/etcd:
+  /boot/v1/service/storage/status:
     get:
-      summary: "Retrieve the current connection status to ETCD"
+      summary: "Retrieve the current connection status to storage backend"
       tags:
       - service-status
       - cli_ignore
       description: |
-        Retrieve the current connection status to the BSS ETCD database.
+        Retrieve the current connection status to the BSS backend storage database. This can
+        either be ETCD or PostgreSQL, depending on which storage backend BSS was started with.
         
-        The connection to ETCD will be tested by writing a value to ETCD, and then reading it
-        back from the database. If the value is successfully writen to ETCD and read back as the 
-        same value, then the connection to ETCD is considered to be connected. Otherwise, there
-        is a connection error.
+        If PostgreSQL is being used, the connection will be tested by attempting to retrieve all
+        of the boot parameters that BSS knows about. If this succeeds (even if the result is
+        empty), then the connection to PostgreSQL is considered to be connected. If this fails,
+        there is a connection error.
+
+        If ETCD is being used, the connection will be tested by writing a value to ETCD, and then
+        reading it back from the database. If the value is successfully writen to ETCD and read
```

Comment:
Small typo: "writen" should be "written"

Reply:
Fixed in 049e330.